.circleci/config.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.3-node
+    enviroment:
+      ACCESSLINT_MASTER_BRANCH: dev
+    steps:
+      - checkout
+      - run: bundle install
+      - run: npm install
+      - run: npm run test
+      - run: npm run htmlproofer
+      - run: bundle exec rspec
+      - run: CODECLIMATE_REPO_TOKEN=${CODECLIMATE_REPO_TOKEN} bundle exec codeclimate-test-reporter
+      - run: ./serve-accesslint && bundle exec accesslint-ci scan http://localhost:4000/site
```

Comment:
Since the accesslint-ci tool has moved to a service, I think we should remove this step and create a new issue to enable the accesslint-ci service instead.